pkg/controller/association/reconciler.go

```diff
@@ -52,8 +59,14 @@ type AssociationInfo struct {
 	AssociationLabels func(associated types.NamespacedName) map[string]string
 	// UserSecretSuffix is used as a suffix in the name of the secret holding user data in the associated namespace.
 	UserSecretSuffix string
+	// CASecretServiceLabelName is the name of the label added on the Secret that contains the CA of the remote service.
+	CASecretServiceLabelName string
```

Comment:
I had to read this a couple of times (CIA Secret, secret service, CIA secret service were among the parse errors my brain made :-) ) when I encountered the name in the code and had to go to the Go doc to understand what the name means ...

Reply:
`CASecretLabelName` is probably good enough and does not involve the FBI?